terraform/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "iam_role_name" {
+  description = "The name of the IAM Role"
+  value       = aws_iam_role.doit-management-role.name
+}
+
+output "iam_role_arn" {
+  description = "The arn of the IAM Role"
+  value       = aws_iam_role.doit-management-role.arn
```

Comment:
@tibor-szalai Don't forget to change the outputs. ;)
```suggestion
  value       = aws_iam_role.doit_management_role.arn
```

Reply:
I did forget. Thanks for reminding me. :)